tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/TsFileMetaData.java

```diff
@@ -110,6 +120,13 @@ public static TsFileMetaData deserializeFrom(InputStream inputStream) throws IOE
     }
     fileMetaData.totalChunkNum = ReadWriteIOUtils.readInt(inputStream);
     fileMetaData.invalidChunkNum = ReadWriteIOUtils.readInt(inputStream);
+    // read bloom filter
+    if(!ReadWriteIOUtils.checkIfMagicString(inputStream)){

```

Comment:
Maybe you can use `available()` to perform a quick check.

Reply:
at least you can not use the name of `checkIfMagicString` because you can only know whether the left is not a magic string, you can not know whether it is a correct magic string...  so, `notMagicString()` may be better than now.